docs/alma/alma.rst

```diff
@@ -294,6 +294,65 @@ their types.
       velocity_resolution  double          m/s        Estimated velocity resolution from all the spectral windows, from frequency resolution.
 
 
+Query Results
+=============
+
+Results of queries are returned in tabular format. For convenience,
+the `~astroquery.alma.get_enhanced_table` function can be used to have the initial result
+in a more useful format, i.e., turn values into quantities, footprint into
+shape, etc. (Note: this require the `regions` Python package to be installed.
+
+
+.. doctest-remote-data::
+
+    >>> from astroquery.alma import Alma, get_enhanced_table
+    >>> alma = Alma()
+    >>> alma.archive_url = 'https://almascience.eso.org'  # optional to make doctest work
+    >>> res = alma.query_tap("select top 1 * from ivoa.ObsCore where obs_publisher_did='ADS/JAO.ALMA#2011.0.00087.S'")
+    >>> enhanced_res = get_enhanced_table(res)
+    >>> enhanced_res[0]['s_ra']
+        <MaskedQuantity 86.82119735 deg>
+    >>> enhanced_res[0]['s_region']
+        <PolygonSkyRegion(vertices=<SkyCoord (ICRS): (ra, dec) in deg
+        [(86.823884, -51.067761), (86.820804, -51.069956),
+         (86.818262, -51.069702), (86.81656 , -51.068489),
+         (86.816655, -51.066594), (86.820904, -51.063002),
+         (86.823701, -51.063002), (86.825834, -51.064343),
+         (86.825915, -51.065959)]>)>
+
+To further draw the footprint:
+
+.. doctest-skip::
```

Comment:
This code example below doesn't generate the footprint image for me. Could we instead use the plot directive to ensure the png and the code remains in sync? (as far as I see plotting this figure doesn't really eat up much resources, so totally can be done on the fly)

Reply:
Not very familiar with plotting. What exactly are you thinking about? BTW, I can plot the image but now it comes as a Circle and I can't find the source of the original one. If we can do it on the fly, then we don't have this problem (footprint of the source changes) anymore.

Reply:
Yes, it was a circle for me, too. And 💯 for all figures and images to be generated on the fly, unless it's some manual labor or eats up too many resources.

There are examples in photutils for `.. plot::` that also shows the source code in the rendered HTML. E.g.: https://raw.githubusercontent.com/astropy/photutils/main/docs/centroids.rst

https://photutils.readthedocs.io/en/stable/centroids.html

Reply:
Ah. It was simple once I've learned about the `plot` directive. Funny thing is that the resulting image is now back to polygon. I think there are actually multiple footprints with different shapes. But the basic plotting concept is now clear.